---
content_type: resource
description: ''
end_time: ''
file: null
learning_resource_types: []
license: https://creativecommons.org/licenses/by-nc-sa/4.0/
ocw_type: ''
optional_tab_title: ''
optional_text: ''
parent_title: Lecture 7
parent_type: CourseSection
related_resources_text: ''
resource_index_text: ''
resourcetype: Video
start_time: ''
title: Exceptions
uid: 3b1d84e0-2b94-31db-a47c-7a089b2f8d84
video_files:
  archive_url: https://archive.org/download/MIT6.0001F16/MIT6_0001F16_Lecture_07_exercise_03_300k.mp4
  video_captions_file: /courses/6-0001-introduction-to-computer-science-and-programming-in-python-fall-2016/3137a1e79cca5702ac65d5f91f630a77_0Whyfs88TYE.vtt
  video_thumbnail_file: https://img.youtube.com/vi/0Whyfs88TYE/default.jpg
  video_transcript_file: /courses/6-0001-introduction-to-computer-science-and-programming-in-python-fall-2016/91d2504ca8c093e5e608930ed5e46535_0Whyfs88TYE.pdf
video_metadata:
  youtube_id: 0Whyfs88TYE
---

**Description:** This in-class question demonstrates the use of exceptions in Python.

**Instructor:** Dr. Ana Bell

